spec/lib/cds_importer/entity_mapper/quota_order_number_mapper_spec.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.describe CdsImporter::EntityMapper::QuotaOrderNumberMapper do
+  it_behaves_like 'an entity mapper' do
+    let(:xml_node) do
+      {
+        'sid' => '21811',
+        'quotaOrderNumberId' => '090718',
+        'metainfo' => {
+          'opType' => 'U',
+          'transactionDate' => '2016-07-27T09:20:17',
+        },
+      }
+    end
+
+    let(:expected_values) do
+      {
+        validity_start_date: nil,
```

Comment:
source data doesn't have validity dates - looks like this is being populated in the database so presumably the xml should have the validity dates in it?